docs/components/checkbox/_spec.scss

```diff
@@ -0,0 +1,113 @@
+@import "components/checkbox/config";
+
+.checkbox-spec {
+  hx-checkbox-control {
+    > label {
+      > hx-checkbox {
+        display: none;
+      }
+    }
+
+    /* ----- PRISTINE ----- */
+    > input[type="checkbox"] {
+      /* imitate :focus */
+      &.mock-focus {
+        // &:extend(hx-checkbox-control > input[type="checkbox"]:focus all);
```

Comment:
❗️ I keep the old LESS &:extends (commented out) to show what I changed.  I'll remove before merging.